common/traits/wc_race_traits.txt

```diff
@@ -2877,10 +2877,12 @@ creature_tortollan = {
 	sea_travel_danger = medium_sea_danger_reduction
 	coastal_sea_travel_danger = medium_sea_danger_reduction
 
+	same_opinion = 10
```

Comment:
why do you keep adding this even tho it doesnt work

Reply:
but I removed it